src/client.ts

```diff
@@ -1157,6 +1158,15 @@ class ConnectionImpl implements Connection {
       } catch (e) {
         conn._abort();
 
+        // in case of ECONNRESET, try again
+        if (typeof e.code === "string" && e.code === "ECONNRESET") {
```

Comment:
I don't think typeof check is needed when you use `===`

Reply:
You are right, that's a mistake. If any check, it should check if `e` is undefined. Anyway I can close this PR, since Elvis recommended to not handle this in the client.